### PR TITLE
[ISSUE #823] Support AdminBrokerProcessor get_broker_config

### LINKED ISSUE #823: [Feature🚀] Support  AdminBrokerProcessor get_broker_config
### Is Your Feature Request Related to a Problem?

_No response_

### Describe the Solution You'd Like

_No response_

### Describe Alternatives You've Considered

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by 847850277**
### Which Issue(s) This PR Fixes(Closes)
Fix #823

Fixes #issue_id

### Brief Description

Support AdminBrokerProcessor get_broker_config

## Summary by CodeRabbit

- **New Features**
  - Introduced functionalities for updating and retrieving broker configurations, enhancing broker management.
  - Added methods to access configuration properties for both broker and message store settings, improving introspection capabilities.
  - Enhanced enums with methods for obtaining string representations of broker roles and flush disk types.
  - Provided access to message store configuration through a new method in the DefaultMessageStore.

- **Bug Fixes**
  - Adjusted control flows in request processing to accurately reflect new functionalities.

- **Documentation**
  - Updated comments for better clarity and readability across various components.

### CODE REVIEW

**Comment by mxsm** [comment]
🚀Thanks for your contribution🎉. CodeRabbit(AI) will review your code first🔥

**Review by mxsm — COMMENTED** [review]
@847850277 I personally think this area can directly use `HashMap<String, String>`, with the key values keeping consistent with the Java version. This is necessary because the Rust version needs to be compatible with the Java version. Otherwise, the configuration retrieved using Java client commands will appear differently. Additionally, as you can see, in the Java version, `BrokerConfig` inherits from `BrokerIdentity`, so after serialization, they should be at the same level. The same should apply in Rust, so the key values should remain consistent.
image

**Comment by mxsm** [comment]
🚀Thanks for your contribution🎉. CodeRabbit(AI) will review your code first🔥

**Comment by 847850277** [comment]
> @847850277 I personally think this area can directly use `HashMap<String, String>`, with the key values keeping consistent with the Java version. This is necessary because the Rust version needs to be compatible with the Java version. Otherwise, the configuration retrieved using Java client commands will appear differently. Additionally, as you can see, in the Java version, `BrokerConfig` inherits from `BrokerIdentity`, so after serialization, they should be at the same level. The same should apply in Rust, so the key values should remain consistent. image

correct.

File: rocketmq-store/src/config/broker_role.rs [hunk 1]
```diff
@@ -27,6 +27,16 @@ pub enum BrokerRole {
...
+impl BrokerRole {
+    pub fn get_broker_role(&self) -> &'static str {
+        match self {
+            BrokerRole::AsyncMaster => "AsyncMaster",
+            BrokerRole::SyncMaster => "SyncMaster",
+            BrokerRole::Slave => "Slave",
+        }
+    }
+}
+
```
Comment (mxsm) [inline rocketmq-store/src/config/broker_role.rs:39]: Due to the differences in naming conventions between Rust and Java, this particular aspect should align with Java's standards.

'''
public enum BrokerRole {
    ASYNC_MASTER,
    SYNC_MASTER,
    SLAVE;
}

'''

File: rocketmq-store/src/config/flush_disk_type.rs [hunk 2]
```diff
@@ -28,6 +28,15 @@ pub enum FlushDiskType {
...
 
+impl FlushDiskType {
+    pub fn get_flush_disk_type(&self) -> &'static str {
+        match self {
+            FlushDiskType::SyncFlush => "SyncFlush",
+            FlushDiskType::AsyncFlush => "AsyncFlush",
+        }
+    }
+}
+
```
Comment (mxsm) [inline rocketmq-store/src/config/flush_disk_type.rs:39]: Due to the differences in naming conventions between Rust and Java, this particular aspect should align with Java's standards.

'''
    SYNC_FLUSH,
    ASYNC_FLUSH

'''

**Review by mxsm — COMMENTED** [review]
@847850277 For serialization, especially concerning enumerations, due to the differing naming conventions between Java and Rust, to ensure compatibility between the two languages, it is necessary to adhere to Java's approach.

**Comment by mxsm** [comment]
🚀Thanks for your contribution🎉. CodeRabbit(AI) will review your code first🔥

**Comment by mxsm** [comment]
🚀Thanks for your contribution🎉. CodeRabbit(AI) will review your code first🔥

**Comment by mxsm** [comment]
🚀Thanks for your contribution🎉. CodeRabbit(AI) will review your code first🔥

**Comment by mxsm** [comment]
🚀Thanks for your contribution🎉. CodeRabbit(AI) will review your code first🔥

**Review by mxsm — APPROVED** [review]
LGTM

**Pull request merged**